import-export-cli/box/resources/sample/api_product_params.yaml

```diff
@@ -0,0 +1,81 @@
+environments:
+    - name:
+      configs:
+          dependentAPIs:
+              <api_1_name>-<api_1_version>:
+                  endpoints:
+                      production:
+                      sandbox:
+                  security:
+                      enabled:
+                      type:
+                      username:
+                      password:
+                  deploymentEnvironments:
```

Comment:
In the current implementation, even though the dependent apis have deployment environments, they will only be deployed in the product's environment. Therefore i don't thing we need to add this to product params files.

Reply:
+1 will remove it from here

Reply:
Fixed via https://github.com/wso2/product-apim-tooling/pull/652/commits/6d527e70824d7f557df5fe9fa2506346f854d9ec